contracts/actions/common/PullToken.sol

```diff
@@ -1,18 +1,22 @@
 pragma solidity ^0.8.15;
 
 import { Executable } from "../common/Executable.sol";
+import { UseStore } from "../common/UseStore.sol";
 import { SafeERC20, IERC20 } from "../../libs/SafeERC20.sol";
 import { PullTokenData } from "../../core/types/Common.sol";
-import { PULL_TOKEN_ACTION } from "../../core/constants/Common.sol";
+import { EVENT_EMITTER, PULL_TOKEN_ACTION } from "../../core/constants/Common.sol";
 import "../../core/types/Common.sol";
+import { IEventEmitter } from "../../interfaces/common/IEventEmitter.sol";
 
 /**
  * @title PullToken Action contract
  * @notice Pulls token from a target address to the current calling context
  */
-contract PullToken is Executable {
+contract PullToken is Executable, UseStore {
```

Comment:
You don't have to implement `UseStore`.
Having `constructor (address _registry)` is just fine.